client/dashboard/section-controls.js

```diff
@@ -36,10 +35,6 @@ class SectionControls extends Component {
 	render() {
 		const { onRemove, isFirst, isLast } = this.props;
 
-		if ( ! window.wcAdminFeatures[ 'analytics-dashboard/customizable' ] ) {
```

Comment:
If we remove this here, could this possibly let a third party add this component to the dashboard even if the `analytics-dashboard/customizable` flag is disabled?

Reply:
Indeed it would, but the filter to add dashboard items is in `customizable.js` which is itself behind the flag.